src/Features/Core/Portable/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -412,36 +412,28 @@ private bool IsCandidateSymbol(ISymbol memberSymbol)
                 if (memberSymbol.DeclaredAccessibility == Accessibility.Private &&
                     !memberSymbol.IsImplicitlyDeclared)
                 {
-                    // Do not track accessors, as we will track the associated symbol.
                     switch (memberSymbol.Kind)
                     {
                         case SymbolKind.Method:
-                            // Skip following methods:
```

Comment:
❓ Can we document the definition of "candidate symbol" explicitly, either at or linked to from the documentation comment for the `IsCandidateSymbol` method?

Reply:
Done.

---
In reply to: [231527481](https://github.com/dotnet/roslyn/pull/30982#discussion_r231527481) [](ancestors = 231527481)